src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIResponseChatClient.cs

```diff
@@ -187,6 +189,8 @@ public async IAsyncEnumerable<ChatResponseUpdate> GetStreamingResponseAsync(
 
             if (streamingUpdate is StreamingResponseItemUpdate itemUpdate)
             {
+                messageId = itemUpdate.Item.Id;
```

Comment:
If you search this file for "msg_ignored", there are a few places these message id could be propagated back to the underlying API (though I think those might go away).

Reply:
Done